api/src/main/java/javax/enterprise/event/Event.java

```diff
@@ -100,6 +100,30 @@
 
     /**
      * <p>
+     * Fires an event with the specified qualifiers and notifies observers.
+     * </p>
+     *
+     * @param event the event object
+     * @throws IllegalArgumentException if the runtime type of the event object contains a type variable
+     * @throws FireAsyncException   if a notified observer throws a checked exception, it will be wrapped and rethrown as an
```

Comment:
This is wrong. An async observer method may throw an exception after this method has already returned.